Arale/AraleHeaderView.swift

```diff
@@ -23,6 +23,13 @@ public class AraleHeaderView: UIView {
      */
     open private(set) var backgroundImage: UIImage?
     
+    /**
+     Gettable image for HeaderView
+     Settable from init method
+     @return the view that is displayed on top of the image view.
+     */
+    open private(set) var overlayView: UIView?
```

Comment:
This is such a great idea! Would be great if we use `contentView` as the name and set an empty UIView as we initialized `AraleHeaderView`. This the same approach as `UITableViewCell` where it has `contentView` that customizable. On top of that too this approach is more familiar to developer who often use UIKit. What do you think?